packages/manager/modules/pci/src/projects/project/edit/edit.html

```diff
@@ -26,23 +27,48 @@
             required
         />
     </oui-field>
+    <oui-field
+        label="{{ ::'pci_projects_project_edit_project_name' | translate }}"
+        label-popover="{{ ::'pci_projects_project_edit_discovery_disable_tooltip' | translate }}"
+        data-ng-if="$ctrl.isDiscoveryProject"
+        size="xl"
+    >
+        <input
+            class="oui-input"
+            type="text"
+            id="description"
+            name="description"
+            data-ng-disabled="$ctrl.isDiscoveryProject"
+            ng-model="$ctrl.project.description"
+            ng-pattern="/^[^<>]+$/"
+            required
+        />
+    </oui-field>
     <oui-field>
         <oui-checkbox
             name="isDefault"
+            disabled="$ctrl.isDiscoveryProject"
             description="{{ ::'pci_projects_project_edit_default_project_description' | translate }}"
             model="$ctrl.isDefault"
             ><span
                 data-translate="pci_projects_project_edit_default_project"
             ></span>
         </oui-checkbox>
     </oui-field>
-    <oui-button variant="primary" type="submit" disabled="$ctrl.loading.submit">
+    <oui-button
+        variant="primary"
+        type="submit"
+        disabled="$ctrl.loading.submit || $ctrl.isDiscoveryProject"
+    >
         {{ ::'pci_projects_project_edit_update' | translate }}
     </oui-button>
 </form>
 
 <!--HDS section-->
-<div class="mt-4 pt-4 border-top" data-ng-if="$ctrl.hds.isAvailable">
+<div
+    class="mt-4 pt-4 border-top"
+    data-ng-if="$ctrl.hds.isAvailable && !$ctrl.isDiscoveryProject"
+>
     <edit-hds-option
```

Comment:
**question**: are you sure that the hds options must be hidden ?

Reply:
Yes, It's mentionned in the JIRA ticket associated to this PR.